src/common/libtaskmap/taskmap.c

```diff
@@ -618,6 +618,9 @@ static char *taskmap_encode_pmi (const struct taskmap *map)
     zlistx_t *l;
```

Comment:
In commit message for "libtaskmap: encode an unknown PMI_..."
s/uknown/unknown/